toolchain/sem_ir/ids.h

```diff
@@ -265,6 +265,11 @@ struct BoolValue : public IdBase, public Printable<BoolValue> {
   static const BoolValue False;
   static const BoolValue True;
 
+  // Returns the `BoolValue` corresponding to `b`.
+  static constexpr auto FromBool(bool b) -> BoolValue {
```

Comment:
Is this used as constexpr?
```suggestion
  static auto FromBool(bool b) -> BoolValue {
```

Reply:
I'm inclined to keep the `constexpr` -- it would be reasonable to use this from constant expressions even though we happen to not use it that way right now.